components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -715,7 +715,7 @@ protected void initFile(String id) throws FormatException, IOException {
       seriesCount *= mosaics;
     }
 
-    ms0.imageCount = getSizeZ() * (isRGB() ? 1 : getSizeC()) * getSizeT();
+    ms0.imageCount = getSizeZ() * (isRGB() ? getSizeC()/3 : getSizeC()) * getSizeT();
```

Comment:
This isn't simply `getEffectiveSizeC()` is it? (I really don't know, I've wondered how all this works!)